exp.Dockerfile

```diff
@@ -1,13 +1,13 @@
 # WEBUI
-FROM node:8.15.0 as webui
+FROM node:12.11 as webui
 
 ENV WEBUI_DIR /src/webui
 RUN mkdir -p $WEBUI_DIR
 
 COPY ./webui/ $WEBUI_DIR/
 
 WORKDIR $WEBUI_DIR
-RUN yarn install
+RUN npm install
```

Comment:
Why the switch from yarn to npm?

Reply:
because the new webui uses npm.
This file was a previous webui relicate.